### ISSUE TITLE
[IA] Determine URL for new page - "Find a VA form"

### ISSUE THREAD

**Issue by ncksllvn** [department-of-veterans-affairs/va.gov-team#3548]
We're replacing https://www.va.gov/vaforms/search_action.asp  with a new modernized application for searching all forms (about 600 in total.) We need to define the URL for where this page will live. For example - www.va.gov/find-va-form.

**Comment by mnorthuis** [comment]
@ncksllvn this work needs a full IA review before URLs can be identified.  I am working with @lwalsh617  on getting this scheduled.

**Comment by aeltarhoni** [comment]
@jenniferlee-dsva had a conversation with Mikki. We have an approach, waiting on recommendation.

**Comment by Karak888** [comment]
need to determine IA piece and how to review.  @Karak888 to follow up with @mnorthuis

**Comment by Karak888** [comment]
@mnorthuis I totally forgot to ask you about this and find out what needs to happen next.  Do I need to fill out a request form or something?  just clueless here so anything you can provide me would be much appreciated.  If yout want to pull up next week that is fine as well.  Thanks @mnorthuis

**Comment by ncksllvn** [comment]
@Karak888 I think this is our request - https://github.com/department-of-veterans-affairs/va.gov-team/issues/3887. Maybe we should add that to our epic?

**Comment by mnorthuis** [comment]
#3887 is for the IA review and will have the URL as part of that, so I think this might be duplicative of that issue.

**Comment by aeltarhoni** [comment]
@ncksllvn Planning to retire search action asp and redirect to https://www.va.gov/vaforms/ and assess keeping https://www.va.gov/vaforms/

**Comment by ncksllvn** [comment]
I suppose we can close this then in favor of https://github.com/department-of-veterans-affairs/va.gov-team/issues/3887, as long as we track that one on our board. What do you think, @Karak888?

**Comment by ncksllvn** [comment]
Closing in favor of https://github.com/department-of-veterans-affairs/va.gov-team/issues/3887
